app/controllers/admin_configurations_controller.rb

```diff
@@ -148,7 +148,7 @@ def manage_firecloud_access
           redirect_to admin_configuration_path, alert: 'Invalid configuration option; ignored.'
       end
     rescue => e
-      ErrorTracker.report_exception(e, current_user, params.to_unsafe_hash)
+      ErrorTracker.report_exception(e, current_user, params)
       logger.error "#{Time.zone.now}: error in setting download status to #{status}; #{e.message}"
```

Comment:
Per [recent discussion](https://github.com/broadinstitute/single_cell_portal_core/pull/1043#discussion_r636325198):

```suggestion
      logger.error "Error in setting download status to #{status}; #{e.message}"
```